src/utils/helper_embedding.py

```diff
@@ -16,3 +18,84 @@ def reshape_df(df: pd.DataFrame, col_name: str) -> pd.DataFrame:
         return df[['word', col_name]]
     except Exception as error:
         print(f"Cause: {error}")
+
+
+def calculate_cosine_similarity(a: list, b: list) -> float:
```

Comment:
Typing is an interesting one, I personally am not much of a fan but it's a personal preference. AFAIK we don't really have an agreed use/no use policy in Data Labs, but I think we ought to do it for production systems. If it helps you then continue doing it and it'll make things easier to translate into production systems 👍